pkgs/top-level/all-packages.nix

```diff
@@ -676,6 +676,8 @@ with pkgs;
 
   glade = callPackage ../development/tools/glade { };
 
+  glance = callPackage ../by-name/gl/glance/package.nix { };
```

Comment:
`pkgs/by-name` is automatic and packages using it do not need to be added to `pkgs/top-level/all-packages.nix`.

Reply:
noticed that from the failed action/check. thanks for confirming.